File: ce/snippets/csharp/CRMV8/query/cs/linqexamples14.cs
// <snippetlinqexamples14>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_equals1 = from c in svcContext.ContactSet
                      where c.FirstName.Equals("Colin")
                      select new
                      {
                       c.FirstName,
                       c.LastName,
                       c.Address1_City
                      };
  foreach (var c in query_equals1)
  {
   System.Console.WriteLine(c.FirstName +
    " " + c.LastName +
    " " + c.Address1_City);
  }
 }

// </snippetlinqexamples14>

File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples14.vb
' <snippetlinqexamples14>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_equals1 = From c In svcContext.ContactSet _
                      Where c.FirstName.Equals("Colin") _
                      Select New With {Key c.FirstName,
                                       Key c.LastName,
                                       Key c.Address1_City}
  For Each c In query_equals1
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName &amp; " " _
                     &amp; c.Address1_City)
  Next c
 End Using

' </snippetlinqexamples14>